README.md

```diff
@@ -1,15 +1,16 @@
 # GitHub Copilot Chat in Visual Studio Code
 
-This repository is for providing feedback on the Copilot Chat UX in VS Code. You can use the repository to report issues or submit feature requests on the chat user experience and interface.
+This repository is for providing feedback on the GitHub Copilot exeperience in VS Code, including both Copilot Chat and completions. You can use this repository to report issues or submit feature requests related to the user experience and interface of either extensions.
 
-GitHub Copilot Chat is a companion extension to GitHub Copilot that houses experimental chat features. To learn more about GitHub Copilot Chat, check out our [blog post](https://code.visualstudio.com/blogs/2023/03/30/vscode-copilot).
+- [GitHub Copilot Chat](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat) is a companion extension to GitHub Copilot that houses experimental chat features.
+- [GitHub Copilot](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) is a auto-complete style experience that provides inline code suggestions as you type, and may include experimental models for code completions.
+
+To learn more about GitHub Copilot, check out our [blog post](https://code.visualstudio.com/blogs/2023/03/30/vscode-copilot).
 
 # Providing Feedback
 
-You can use this repository to file issues on the UX and UI for Copilot Chat in VS Code:
+You can use this repository to file issues on the UX and UI for Copilot Chat and completions in VS Code:
```

Comment:
Should we explicitly say "UX and UI"? If users aren't filing a UX or UI issue, where do they go?

Reply:
Good point :)